source/howto/mobility/getstarted.rst

```diff
@@ -123,33 +131,107 @@ incrementally refine the data retrieved. They are:
 
-.. topic:: Showing and browsing data with API v2
-	   
-   The first method described in the previous list introduces a new
-   facility made available by the API v2: the type of
-   `representation` that can be used to browse or access the data
-   provided by the Open Data Hub Team.
-
-   The three alternative, which will be described in detail in an
-   upcoming howto, are: `flat`, `tree`, and `apispec`.
-
-   The `flat` and the `tree` alternatives are :term:`JSON`
-   representation of the data, whereas `apispec` is a YAML
-   representation in OpenAPI v3 format suitable for swagger-like
-   access to the data.
-
-   In both the :strong:`flat` and :strong:`tree` representations, all
-   the metadata and available data are shown and browsable, the
-   difference being that in `flat` all metadata are shown at the same
-   level, while `tree` keeps the hierarchical structure of the
-   metadata. Both of them are available
+Representation types
+~~~~~~~~~~~~~~~~~~~~
    
-   The :strong:`apispec` is a YAML configuration file that can be used
-   to set up a swagger-like interface to the Open Data Hub's data.
+.. versionchanged:: 2020.10 New types of representation for API calls:
+   node and edge.
+
+The first method described in the previous list introduces the
+available entry points to the API v2: the types of `representation`
+that can be used to browse or access the data provided by the Open
+Data Hub Team
+
+The `representation` consists now of a pair of comma-separated
+keywords composed of:
+
+1. the already existent `flat` or `tree` AND
+2. either `node` and `edge`
+
+In both the :strong:`flat` and :strong:`tree` representations, all the
+metadata and available data are shown and browsable, the difference
+being that in `flat`, while
+`tree` keeps the hierarchical structure of the metadata.
 
+The `node` and `edge` describe a :literal:`StationType` and the
+connection between two :literal:`StationType`\s, respectively.
+
+
+.. _edge-definition:
+
+.. _node-definition:
+
+.. panels::
+
+   Flat
+   ^^^^
+   
+   In the `flat` representation, all metadata and available data can
+   be accessed and browsed. However, no hierarchy appears and data and
+   metadata are shown at the same level.
+
+   ----
+   
+   Tree
+   ^^^^^
+
+   In the `tree` representation, all metadata and available data can
+   be accessed and browsed as in `flat`, but in this case, any
+   hierarchy of data or metadata is preserved and shown.
+
+   
+   -----
+   Node
+   ^^^^^
+
+   A node is a measurement station and contains all metadata
+   associated to it. The :strong:`node` representation corresponds to
+   the `old` (pre-2020.10) output of the API calls, therefore it can
+   safely be omitted for backward compatibility. As an example, valid
+   for all methods listed in the :ref:`previous section
+   <api-v2-structure>`, these two API calls are equivalent.
+
+     :literal:`/v2/node/{stationTypes}`
+	    
+     :literal:`/v2/{stationTypes}`
```

Comment:
This is not correct, since flat or tree must always be given, because both examples would produce an error

Reply:
oh right, it should be `/v2/flat,node/{stationTypes}` or `/v2/flat/{stationTypes}` or even  `/v2/edge/{stationTypes}` then

Reply:
The first two examples are correct, the last not.. edge must be combined with flat or tree ;-)

Reply:
@#£$%&& yes, sorry... the last would be `/v2/tree/{stationTypes}`

Reply:
@Piiit I should have gotten it now :-)